test/integration/suites/node-attestation/03-test-node-attestation

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+# Test node attestation api
+jointoken=`docker-compose exec -u 1001 -T spire-server /opt/spire/conf/server/node-attestation -testStep jointoken`
```

Comment:
instead using a new client, why not to use regular server CLI? we need it only to create a token